src/main/kotlin/de/bund/digitalservice/useid/identification/IdentificationSessionRepository.kt

```diff
@@ -10,5 +11,9 @@ interface IdentificationSessionRepository : JpaRepository<IdentificationSession,
     fun findByEIdSessionId(eIdSessionId: UUID): IdentificationSession?
     fun findByUseIdSessionId(useIdSessionId: UUID): IdentificationSession?
     fun deleteAllByCreatedAtBefore(before: LocalDateTime)
+
+    @Modifying
+    @Query("DELETE FROM IdentificationSession session WHERE session.eIdSessionId = :eIdSessionId")
+    fun deleteByEIdSessionId(eIdSessionId: UUID)
```

Comment:
It's not the same thing. This part here removes identification sessions from the database which are completed successfully and therefore not needed anymore. The schedular removes idle session which have been started but never completed. 
I would still vote for deleting the completed once immediately since there is no reason to keep them.